### DISCUSSION TITLE
So which to install for V28?

### DISCUSSION THREAD

**Discussion by OGNetheri** [DistroAV/DistroAV#781] [Q&A - Support]
So just a stupid(?) question: Which version to install on Windows10/11, "obs-ndi-4.10.0-Qt6-Windows-Installer.exe" or "obs-ndi-5.0.0-windows-x64-Installer.exe" from the version 4.10.0 assets? 

Also, does the officially V28 version update the sources from dev47 forked v28 compatability version to NDI5 versions in OBS, or does these keep the NDI 4 version in use or does eitther of these installer or both upgrade only so one must 1st uninstall forked 4.9.2. and make all the NDI's from a scratch were they version 4 or 5?

So basically install instructions would be nice especially in the case where users had to install forkerd versions od the 4.9.1 (dev47apps's 4.9.2) and also mention that if the 4.1.0. is ONLY compatibility update without the new sound system (like the dev47apps's 4.9.2), is there any need to install 4.10.0 exe or the 5.0 exe?!

Edit: Thanks for the instructions/desciption in the releases. It made clear there's no need to install 4.10.0, if one uses Win10/11 version of Dev47apps forked 4.9.2.  ( https://github.com/dev47apps/obs-ndi/releases/tag/4.9.2 )

Hopefully the upcoming 5.0 version of the Palakis's OBS-NDI can update/import the 4.9.1. (V28 4.10.0) based NDI sources&inputs to NDI5 versions without making everything again, it would really save lot of time on native PC2PC NDI rigs.

**Comment by Jaybonaut** [comment]
Also, is there a difference between the 'dummy' release and current?  I installed the 'dummy' version when that went up and it mentioned grabbing the NDI 5 runtime

**Comment by matiaspl** [comment]
At first glance 5.0 lacks the filter function (only PGM and PVW output is available, no separate scenes or sources). The dummy 4.x release adds support for v28 by using the new QT libraries, and so it should work no worse than the last version on v27.

**Comment by crashworldtk** [comment]
I am rather confused as well.

**Comment by Jib-K** [comment]
i saw that there was an update to the ndi so I finally updated both PCs to v28 and updated both the NDI plugins thinking FINALLY! Right? Nope, still doesn't work. Don't know if I'm doing something wrong. Previously I had rolled everything back to get it to work but OBS has been crashing on the stream PC more often than not. So i wanted to update to see if it could smooth it out cuz my streams have been freezing a lot.

**Comment by Trouffman** [comment]
Closing old thread.
Reason : Using an old plugin/OBS version or old-issue.
